.gitignore

```diff
@@ -39,8 +37,6 @@ webrev
 opengrok-web-nbproject/nbproject/private/
 opengrok-web-nbproject/nbproject/genfiles.properties
 run_my.bat
-*.xml~
-opengrok-web-nbproject/nbproject/build-impl.xml~
 build.bat
```

Comment:
Why are some changes in this file needed?

Reply:
I added an entry to ignore vim's .swp files and also noticed the redundancy with entries for files ending with "~" already excluded by line 1.

Reply:
ok